x/sequencer/keeper/msg_server_create_sequencer.go

```diff
@@ -33,13 +32,6 @@ func (k msgServer) CreateSequencer(goCtx context.Context, msg *types.MsgCreateSe
 		return nil, types.ErrRollappJailed
 	}
```

Comment:
don't u need to check the initial_genesis_sequencer instead?

Reply:
also we probably should check that the checksum is defined as well. can't have proposer sequencer if no checksum is committed

Reply:
this is to be done in the register sequencer PR

Reply:
not sure why. it's part of this feature

Reply:
I think `CreateSequencer` changes should be done in the PR related to the new create sequencer feature. In the end, the other branch merges into this one

Reply:
changes made since your comment